### PR TITLE
Move openshift-priv to app.ci

### PR DESCRIPTION
**Pull request by hongkailiu**
Blocking https://github.com/openshift/release/pull/17254

/cc @stevekuznetsov @droslean

### CODE REVIEW

File: clusters/app.ci/supplemental-ci-images/openshift-priv/admin_roles.yaml [hunk 1]
```diff
@@ -32,9 +32,9 @@ items:
     name: image-tagger-ci-operator
     namespace: ocp-private
   roleRef:
+    apiGroup: rbac.authorization.k8s.io
     kind: Role
     name: image-tagger
-    namespace: ocp-private
```
Comment (droslean) [inline clusters/app.ci/supplemental-ci-images/openshift-priv/admin_roles.yaml:37]: why is this removed?

Comment (hongkailiu) [reply clusters/app.ci/supplemental-ci-images/openshift-priv/admin_roles.yaml:37]:
```
$ oc apply -f  clusters/app.ci/supplemental-ci-images/openshift-priv/admin_roles.yaml --dry-run=server --as system:admin --validate=true
error: error validating "clusters/app.ci/supplemental-ci-images/openshift-priv/admin_roles.yaml": error validating data: [ValidationError(RoleBinding.roleRef): unknown field "namespace" in io.k8s.api.rbac.v1.RoleRef, ValidationError(RoleBinding.roleRef): missing required field "apiGroup" in io.k8s.api.rbac.v1.RoleRef]; if you choose to ignore these errors, turn validation off with --validate=false
```

I think it is has to be the role in the same namespace as the rolebinding.

**Review by droslean — APPROVED** [review]
